driver-dse/src/main/java/com/datastax/driver/graph/GraphStatement.java

```diff
@@ -0,0 +1,135 @@
+            return;
+        }
+        this.JsonParams.clear();
+        try {
+            for (Map.Entry<String, Object> param : this.valuesMap.entrySet()) {
+                StringWriter stringWriter = new StringWriter();
+                JsonGenerator generator = jsonFactory.createGenerator(stringWriter);
+                ObjectNode parameter = factory.objectNode();
+                String name = param.getKey();
+                Object value = param.getValue();
+                if (value instanceof Integer) {
+                    parameter.put("name", name);
+                    parameter.put("value", (Integer)value);
+                } else if (value instanceof String) {
+                    parameter.put("name", name);
+                    parameter.put("value", (String)value);
+                } else if (value instanceof Float) {
+                    parameter.put("name", name);
+                    parameter.put("value", (Float)value);
+                } else if (value instanceof Double) {
+                    parameter.put("name", name);
+                    parameter.put("value", (Double)value);
+                } else if (value instanceof Boolean) {
+                    parameter.put("name", name);
+                    parameter.put("value", (Boolean)value);
+                } else if (value instanceof Long) {
+                    parameter.put("name", name);
+                    parameter.put("value", (Long)value);
+                } else {
+                    throw new DriverException("Parameter : " + value + ", is not in a valid format to be sent as Gremlin parameter.");
+                }
+                objectMapper.writeTree(generator, parameter);
+                this.JsonParams.add(stringWriter.toString());
+            }
+            this.paramsHash = this.valuesMap.hashCode();
+        } catch (IOException e) {
+            throw new DriverException("Some values are not in a compatible type to be serialized in a Gremlin Query.");
+        }
+    }
+
+    @Override
+    SimpleStatement configureAndGetWrappedStatement() {
+        if (hasValues()) {
+            processValues();
+        }
+        this.wrappedStatement = session.getSession().newSimpleStatement(this.query, this.JsonParams.toArray());
+        configure();
+        return this.wrappedStatement;
+    }
+
+    /**
+     * API
+     */
+
+    /**
+     * Set a parameter value for the statement.
+     *
+     * Values can be any type supported in JSON.
+     * @param name Name of the value, defined in the query. Parameters in Gremlin are named as variables, no
+     *             need for a CQL syntax like the bind marker "?" or the identifier ":" in front of a parameter.
+     *             Please refer to Gremlin's documentation for more information.
+     * @param value Any object serializable in JSON. The type will be detected automatically at statement's execution.
+     */
+    public void set(String name, Object value) {
```

Comment:
I tried that and it does work because `getId().toString()` returns the json payload (i.e.: ```{member_id: "0", community_id: "800066", ~label: "comic", group_id: 
"1"}```) which gremlin doesn't like for some reason (maybe that is something that should work on the server side?)